client/src/components/LinkComponent/LinkComponent.scss

```diff
@@ -0,0 +1,10 @@
+@import '@/variables';
+
+.link-component {
+  color: $green;
+  text-decoration: underline;
+
+  &:hover {
+    text-decoration: underline;
+  }
```

Comment:
What's the purpose of this hover?

Reply:
I moved underline from hover and forgot to remove it